lcs/strategies/action_planning.py

```diff
@@ -0,0 +1,258 @@
+from lcs import Perception
+from lcs.agents.acs2 import ClassifiersList
+
+
+class GoalSequenceSearcher:
```

Comment:
Jeśli w pliku jest jedna klasa to jego nazwa powinna być analogiczna.
Proponuję taką strukturę.

```
lcs
  strategies
    action_planning
      action_planning.py
      goal_sequence_searcher.py
```